containers/docker-compose.yaml

```diff
@@ -82,6 +83,7 @@ services:
       args:
         REACT_APP_GDS_API_ENDPOINT: http://localhost:4434/
         REACT_APP_GDS_IS_TESTNET: "true"
+        REACT_APP_GDS_CLIENT_ID: "TODO"
```

Comment:
I need to Google how to set an environment variable as a build arg in docker compose.

Reply:
This might help? https://stackoverflow.com/a/52430444/6552250

Reply:
That did it - thanks!